proxyd/backend.go

```diff
@@ -349,7 +349,17 @@ func (b *Backend) setOffline() {
 }
 
 func (b *Backend) doForward(ctx context.Context, rpcReqs []*RPCReq, isBatch bool) ([]*RPCRes, error) {
-	body := mustMarshalJSON(rpcReqs)
+	isSingleElementBatch := len(rpcReqs) == 0
```

Comment:
shouldn't this be `len(rpcReqs) == 1`?

Reply:
You are absolutely right. Nice catch, this was sloppy on my part.

Reply:
Fixed.